azkaban-web-server/src/main/java/azkaban/webapp/servlet/WebUtils.java

```diff
@@ -108,4 +111,27 @@ public static String getRealClientIpAddr(final Map<String, String> httpHeaders,
     return clientIp;
   }
 
+  /**
+   * Gets the actual client IP address inspecting the X-Forwarded-For HTTP header or using the
```

Comment:
nit: Getting the actual IP address is best effort, it's not guaranteed as the client can sit behind a VPN. Could the comment reflect that?